notification/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2 on 2023-05-24 12:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('player', '0002_static_players'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Notification',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('seen_time', models.DateTimeField(blank=True, null=True)),
+                ('sent_time', models.DateTimeField(default=django.utils.timezone.now)),
+                ('message', models.TextField(blank=True, null=True)),
```

Comment:
Can you explain me the purpose of this field? If I understand correctly this line, you created 'message' text field. Consider maybe to use the message model we have already created, because his purpose is to store text message,
but maybe I don't understand the purpose of this field here.

Reply:
This shopuld be an text field for the notification.
I dont understand ehat are you asking me. 
Maybe to use the message model we have instead this field?

Reply:
> This should be a text field for the notification. I dont understand what are you asking me. Maybe to use the message model we have instead this field?

Ok, now I'm understanding what you meant.
You don't need to do what I said, it's great like that.

Reply:
Ok, but if you see some things like that its good.
Maybe its better to make a connection between messeges and notfication or seperate them like that.
Now I think its good the make it like that seperatly.